openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -12,6 +12,7 @@
 from django.urls import reverse
 from django.utils import timezone
 from django.contrib.auth import get_user_model
+from edx_toggles.toggles.testutils import override_waffle_flag
 from pyquery import PyQuery as pq
 from xmodule.modulestore.tests.django_utils import (
     TEST_DATA_MONGO_AMNESTY_MODULESTORE, ModuleStoreTestCase, SharedModuleStoreTestCase,
```

Comment:
Do you still need TEST_DATA_MONGO_AMNESTY_MODULESTORE?

Reply:
Yeah another test in this file uses it still. pylint would yell at me if it was a dead import.